evcxr_repl/src/bin/evcxr.rs

```diff
@@ -52,6 +52,7 @@ fn send_output<T: io::Write + Send + 'static>(
 
 impl Repl {
     fn new(ide_mode: bool) -> Result<Repl, Error> {
+        let _ret = control::set_virtual_terminal(true);
```

Comment:
Thanks for the fix! The [docs](https://docs.rs/colored/1.9.3/x86_64-pc-windows-msvc/colored/control/fn.set_virtual_terminal.html) say this  function is only available on windows, so you probably need to use `#[cfg(windows)]` to make sure this line is excluded on other platforms.

Reply:
You are right! I knew I was missing something, thank you for the heads up!

Reply:
@davidlattimore Is it fine if I use the cfg! macro? Otherwise I think I would need to make a separate function for it. Which approach would you prefer?